cfme/tests/control/test_actions.py

```diff
@@ -718,13 +718,13 @@ def test_action_untag(request, vm, vm_off, policy_for_testing):
     """
     if not any(tag.category.display_name == "Service Level" and tag.display_name == "Gold"
            for tag in vm.crud.get_tags()):
-        vm.crud.add_tag(("Service Level", "Gold"), single_value=True)
+        vm.crud.add_tag("Service Level", "Gold")
```

Comment:
**QUESTION:** Just a note, do we know why one function requires ```tag``` arg as a kwarg, and the other doesn't?

Reply:
yes, we can use 2 cases, first one to pass category and tag as strings, and pass a tag object, for an object we need to specify a parameter, we have this in docs